hikari/impl/stateful_cache.py

```diff
@@ -792,7 +822,7 @@ def _clear_emojis(
     ) -> cache.CacheView[snowflake.Snowflake, emojis.KnownCustomEmoji]:
         emoji_ids: typing.Iterable[snowflake.Snowflake]
         if guild_id is undefined.UNDEFINED:
-            emoji_ids = tuple(self._emoji_entries.keys())
+            emoji_ids = _copy_mapping(self._emoji_entries).keys()
```

Comment:
Do you even need a key view here? Can you just iterate over the dict directly?

Reply:
yeah it'll work without, missed those when i was removing redundant .keys calls and will remove